addons/web/static/src/core/datetime/datetime_picker_popover.xml

```diff
@@ -3,10 +3,15 @@
     <t t-name="web.DateTimePickerPopover" owl="1">
         <DateTimePicker t-props="props.pickerProps">
             <t t-set-slot="buttons">
-                <t t-if="props.pickerProps.type === 'datetime' or Array.isArray(props.pickerProps.value)">
-                    <button class="o_apply btn btn-primary btn-sm w-100 w-md-auto" t-on-click="props.close">
-                        <i class="fa fa-check pe-1" />
-                        <span>Apply</span>
+                <t
+                    t-if="props.pickerProps.type === 'datetime' or Array.isArray(props.pickerProps.value)"
+                >
+                    <button
+                        class="o_apply btn btn-primary btn-sm w-100 w-md-auto d-flex align-items-center justify-content-center gap-1"
+                        t-on-click="props.close"
+                    >
+                        <i class="fa fa-check" />
+                        Apply
```

Comment:
Is the span element removal necessary ?
Again, if you can leave it we could avoid potential breakage in some eventual customization.